swagger-spec/schema_response_actions/definition.yaml

```diff
@@ -0,0 +1,92 @@
+type: object
+title: Schema Response Actions
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: 'The unique identifier of the Schema Response Action.'
+  type:
+    type: string
+    readOnly: true
+    description: 'The type identifier of the Schema Response Action (`schema-response-actions`).'
+  attributes:
+    type: object
+    title: Attributes
+    readOnly: true
+    properties:
+      trigger:
+        enum:
+        - submit
+        - accept_submission
+        - reject_submission
+        - request_withdrawal
+        - accept_withdrawal
+        - reject_withdrawal
+        - force_withdrawal
```

Comment:
`submit`, `approve`, `admin_reject`, `accept`, and `moderator_reject` seem like the triggers for SchemaResponseActions